### PR TITLE
#FEELTHEGOOF Election PR: This is what the local conservative radio said all sanders voters wanted

### PR DESCRIPTION
**Pull request by Iamgoofball**
**uploading a video now, it looks better in game trust me**
:cl: Goofball Sanders
add: Added a new plant to the game to encourage the Chef to be a more relevant job, alongside a recipe for a food related to it. Throw a cannabis leaf on a table and check the recipes list to find it.
/:cl:

Feedback:
dreamseeker_2016-07-03_01-59-46

usinng my token from https://github.com/tgstation/tgstation/pull/18170 since it still hasnt been successfully used

### CODE REVIEW

**Comment by InsaneHyena** [comment]
Must have a Trump PR as a counterweight. RCD wall building buffs when?

**Comment by Iamgoofball** [comment]
@insanehyena sorry the email plans for that got leaked and we had to go fundraise for new ones

**Comment by MrPerson** [comment]
All the weed should go in the goon folder

**Comment by Iamgoofball** [comment]
oh yeah, forgot we separate the sprites, will move them

**Comment by evsey9** [comment]
make a pr that makes you headmin, it would get merged, totally, i swear

**Comment by coiax** [comment]
Right, I had to reread this multiple times to determine this was a PR adding cannabis to the game, the memes are thick and difficult to decipher.

**Comment by X-TheDark** [comment]
Soon enough Goof will make a Lipocide Storm event that puts it into all mobs that aren't in maintenance to make the Chef relevant.

File: code/modules/hydroponics/grown/cannabis.dm [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+						"methamphetamine" = 0.15,
+						"capsaicin" = 0.15,
+						"barbers_aid" = 0.15,
+						"bath_salts" = 0.15,
+						"itching_powder" = 0.15,
+						"crank" = 0.15,
+						"krokodil" = 0.15,
+						"histamine" = 0.15,
+						"lipolicide" = 0.15)
+	rarity = 420
```
Comment (Supermichael777) [inline code/modules/hydroponics/grown/cannabis.dm]: your memes will force dreams

Comment (X-TheDark) [inline code/modules/hydroponics/grown/cannabis.dm]: How many cargo points would this even be worth?

**Comment by Fox-McCloud** [comment]
Needed to happen :+1:

**Comment by optimumtact** [comment]
What's the point of having this and ambrosia?

**Comment by Iamgoofball** [comment]
ambrosia is primarily a medical plant when it comes to gameplay, with each mutation level bumping up the healing power and downsizing on the space drugs. If anything, I think the space drugs in ambrosia was actually to try to be a downside to the healing meds.

**Comment by Iamgoofball** [comment]
these new plants have the munchies mechanic and will ensure the chef continues to be useful to those who don't think about what they're doing

**Comment by KorPhaeron** [comment]
none

**Comment by JJRcop** [comment]
It seems like the different cannabis should have different effects.

**Comment by JJRcop** [comment]
i read the code wrong

**Comment by Firecage** [comment]
Eh, I am a bit conflicted on Death Weed containing Cyanide.

**Comment by Iamgoofball** [comment]
its deadly

**Comment by Firecage** [comment]
I do understand that goof, but I woulda said cyanide is a bit over the top, but browsing the code recently, I don't know what the fuck happened. Poison Apples no longer contains cyanide but is still admin only. Poison berries now contains cyanide, but their more lethal mutation Death berries doesn't....so...ehh...

Have fun Goof!!!!

**Comment by Iamgoofball** [comment]
i thought poison berries had some other shit in them? Hm, i'll have to check them out

**Comment by Firecage** [comment]
Well, yah, used to contain normal toxin. Welp, now I see what happened, bloody Lumi.

Anywho, good PR anyway goof. If you get onto IRC later, I guess ile link you to said PR which changed the chemical reagents in plants.

**Comment by Supermichael777** [comment]
I didnt know nazis leaned sanders /s

**Comment by Cheridan** [comment]
DUDE WEED LMAO

**Comment by bobdobbington** [comment]
Cyanide is a meme now, it's basically a much shittier version of Lexorin, it's not anywhere near the level of combat robustness it had back when every nuke op got issued a suicide pill made of cyanide.

**Comment by Iamgoofball** [comment]
I'll swap out the toxin.

**Comment by Firecage** [comment]
@Iamgoofball But as I said in OOC. I do think you should probably add some practical uses to cannibas so it isn't just another random botany plant added for memes.

One or two kitchen recipes would work. Would also be nice if you added the ability to craft rope from cannibas plants and then added some use for them.

**Comment by X-TheDark** [comment]
Hash cakes when?

**Comment by optimumtact** [comment]
Add a food recipe and I'll merge it

**Comment by Iamgoofball** [comment]
fine

**Comment by Iamgoofball** [comment]
ALSO I have to move the plant sprites back to the base hydroponics growing icons, the code doesn't support grabbing from other icon files and its dumb

File: code/modules/hydroponics/grown/cannabis.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/hydroponics/grown/cannabis.dm]: also set this to a reasonable number

File: code/modules/hydroponics/grown/cannabis.dm [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+/obj/item/weapon/reagent_containers/food/snacks/grown/cannabis/white
+	seed = /obj/item/seeds/cannabis/white
+	desc = "It feels smooth and nice to the touch."
+	icon_state = "whitecannabis"
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/cannabis/ultimate
+	seed = /obj/item/seeds/cannabis/ultimate
+	desc = "You feel dizzy looking at it. What the fuck?"
+	icon_state = "ocannabis"
+	volume = 420
```
Comment (optimumtact) [inline code/modules/hydroponics/grown/cannabis.dm]: again reasonable number please

Comment (Iamgoofball) [inline code/modules/hydroponics/grown/cannabis.dm]: it's actually kind of reasonable for super high potency seeds

File: code/modules/hydroponics/grown/cannabis.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/hydroponics/grown/cannabis.dm]: rare mutation but OK

**Comment by Iamgoofball** [comment]
@optimumtact we're ready

**Pull request merged**